docs/topics/native/native-definition-file.md

```diff
@@ -0,0 +1,266 @@
+| [`noStringConversion`](#set-up-string-conversion)                                   | A space-separated lists of functions whose `const char*` parameters should not be auto-converted to Kotlin `String`s.                                                                                                       |
+| `allowedOverloadsForCFunctions`                                                     | By default, it's considered that C functions have unique names. If several functions have the same name, only one is picked. However, you can change this by specifying these functions in `allowedOverloadsForCFunctions`. |
+| [`disableDesignatedInitializerChecks`](#allow-calling-a-non-designated-initializer) | Disables the compiler check that doesn't allow calling a non-designated Objective-C initializer as a `super()` constructor.                                                                                                 |
+| [`foreignExceptionMode`](#handle-objective-c-exceptions)                            | Wraps exceptions from Objective-C code into Kotlin exceptions with the `ForeignException` type.                                                                                                                             |
+| `objcClassesIncludingCategories`                                                    |                                                                                                                                                                                                                             |
+| [`userSetupHint`](#help-resolve-linker-errors)                                      | Adds a custom message, for example, to help users resolve linker errors.                                                                                                                                                    |
+
+In addition to the list of properties, you can include [custom declarations](#add-custom-declarations) in your definition
+file.
+
+### Import headers
+
+If a C library does not have a Clang module and instead consists of a set of headers, use the `headers` property to
+specify headers that should be imported:
+
+```none
+headers = curl/curl.h
+```
+
+#### Filter headers by globs
+
+You can filter headers by globs using filter properties from the `.def` file. To include declarations from headers,
+use the `headerFilter` property. If a header matches any of the globs, its declarations are included in the bindings.
+
+The globs are applied to the header paths relative to the appropriate include path elements,
+for example, `time.h` or `curl/curl.h`. So if the library is usually included with `#include <SomeLibrary/Header.h>`,
+it would probably be correct to filter headers with the following filter:
```

Comment:
```suggestion
you can filter headers with the following filter:
```
Trying to avoid the conditional but let me know if my suggestion doesn't work for you :)

Reply:
I think it's too confident compared to the original, altered a bit